src/classes/TestResult.js

```diff
@@ -361,6 +382,13 @@ ${ this.error.stack }`);
 		let test = this.test;
 		let ret = {pass: true, details: []};
 
+		if (this.error && this.error.message.includes("Timed out")) {
```

Comment:
I’m not sure we should be handling these as errors. Conceptually, a timeout and an error are different things. 🤔